src/parser/jobs/ast/modules/CelestialOpposition.tsx

```diff
@@ -47,40 +55,44 @@ export default class CelestialOpposition extends Module {
 		this.lastUse = event.timestamp
 	}
 
-	private onSect(event: BuffEvent) {
-		this.activeSect = getDataBy(this.data.statuses, 'id', event.ability.guid)
+	private onSect(event: Events['statusApply']) {
+		if (event.status === this.data.statuses.DIURNAL_OPPOSITION.id) {
+			this.activeSect = this.data.actions.DIURNAL_SECT.id
+		} else {
+			this.activeSect = this.data.actions.NOCTURNAL_SECT.id
+		}
```

Comment:
`activeSect` is a status, so just set it to `event.status` and be done with it. Mixing and matching actions here is a bad idea.

Reply:
the original code had it using actions. I must've mixed them up with the other comments.
it has been updated to just use status instead